### DISCUSSION TITLE
Weekly scheduled backups/updates

### DISCUSSION THREAD

**Discussion by ghost** [nextcloud/all-in-one#483] [Ideas] [answered]
Increase scheduled backups/updates from daily to weekly setting.  E.g. 4 a.m. on Sunday.

**Comment by szaimen** [comment]
Moving to discussions as this is rather a new feature.

**Comment by szaimen** [comment]
Why do you need this feature?

**Reply by ghost** [reply]
So that backups don't take up too much storage space. And to minimize server load.

**Reply by szaimen** [reply]
Did you see this and below?
https://github.com/nextcloud/all-in-one/discussions/428#discussioncomment-2496070

**Reply by ghost** [reply]
Yes I have seen that,
Even though Borg is very efficient in storage, many want to reduce the server load (for home servers the power costs)

**Reply by szaimen** [reply]
> many want to reduce the server load (for home servers the power costs)

I mean I haven't tested the server load but the load should be not higher during the backup because we shut-down all containers before starting the backup and borg is only able to use one process currently which means that the load should not really increase (or increase very slightly). Also because only changes will be backed up, it is a matter of max. a few minutes usually for the whole process.

**Comment by szaimen** [comment, marked as answer]
Honestly, I don't see a good usecase for this option. You should rather be happy when something happens to your files/data and the last backup is not older than 24h...
